lib/sensors/i2c_mux.hpp

```diff
@@ -10,19 +10,23 @@
 
 namespace hyped::sensors {
 
-static constexpr std::uint8_t kDefaultMuxAddress = 0x70;
-static constexpr std::uint8_t kMaxNumChannels    = 8;
-static constexpr core::Float kFailureThreshold
-  = 0.25;  // TODOLater: finalize this value with Electronics
+constexpr std::uint8_t kDefaultMuxAddress = 0x70;
+constexpr std::uint8_t kMaxNumChannels    = 8;
+constexpr core::Float kFailureThreshold = 0.25;  // TODOLater: finalize this value with Electronics
```

Comment:
what does the failure threshold represent? a percentage? or some other value?

Reply:
percentage - courtesy of Bas

Reply:
stick that in a comment